wrapper.go

```diff
@@ -32,10 +34,22 @@ const (
 func init() {
 	logger = log.New()
 	logger.Out = os.Stdout
-	logger.Formatter = &easy.Formatter{
-		TimestampFormat: "2006-01-02 15:04:05",
-		LogFormat:       "#LUMIGO# - %time% - %lvl% - %msg%\n",
+	logger.Formatter = &LogFormatter{}
+}
+
+//Log custom format
+type LogFormatter struct{}
+
+//Format details
+func (s *LogFormatter) Format(entry *log.Entry) ([]byte, error) {
+	timestamp := time.Now().Local().Format("2006-01-02 15:04:05")
+	msg := fmt.Sprintf("#LUMIGO#-%s-%s-message:%s ", timestamp, strings.ToUpper(entry.Level.String()), entry.Message)
```

Comment:
Are you sure that it keeps the previous format? Can you send me an image of an example of the new log?

Reply:
#LUMIGO# 2022-04-14 16:27:22-ERROR message:unable to fetch Amazon Trace ID

Reply:
![image](https://user-images.githubusercontent.com/49228804/163408293-a4df27f1-f657-4f58-bbdc-aaddb1308ab5.png)
We don't have the "message" in the other tracers and it i looks a bit different, check the image